### PR TITLE
Fix default health bar having a very weird anchor point in the skin editor

### LINKED ISSUE #22411: Origin of `DefaultHealthDisplay` skin component behaves abnormally
### Type

Game behaviour

### Bug description

Unlike every other skin component, the origin point of `DefaultHealthDisplay` completely detaches from the component itself, even without resizing (although resizing allows the bug to be displayed more prominently.)

### Screenshots or videos

image

https://user-images.githubusercontent.com/37262651/214528539-562ad0f6-c66e-4619-a85f-f2688aaf0ff3.mp4

### Version

master build (commit efe57f2)

### Logs

[database.log](https://github.com/ppy/osu/files/10498025/database.log)
[input.log](https://github.com/ppy/osu/files/10498026/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/10498027/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/10498028/network.log)
[performance.log](https://github.com/ppy/osu/files/10498029/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10498030/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10498031/updater.log)

[client.realm.zip](https://github.com/ppy/osu/files/10498033/client.realm.zip)

### PR DESCRIPTION
**Pull request by peppy**
Closes https://github.com/ppy/osu/issues/22411.

### CODE REVIEW

File: osu.Game/Screens/Play/HUD/DefaultHealthDisplay.cs
Comment (bdach) [inline osu.Game/Screens/Play/HUD/DefaultHealthDisplay.cs:89]: Applying the padding on both sides has a small side effect, namely that the mod icons (and playback controls, if in replay) get shifted down a tiny bit:

| master | this PR |
| :-: | :-: |
| osu_2023-01-26_20-46-03 | osu_2023-01-26_20-46-51 |

I don't think it matters in the slightest since these controls weren't really aligned to anything properly anyways, but I will wait for a second approval and/or confirmation from @peppy that this is A-OK before clicking the button.

**Review by bdach — APPROVED** [review]

File: osu.Game/Screens/Play/HUD/DefaultHealthDisplay.cs
Comment (peppy) [reply osu.Game/Screens/Play/HUD/DefaultHealthDisplay.cs:89]: I think that's okay for now, yeah. I think I prefer the slightly larger padding.

**Pull request merged**
